pages/index.vue

```diff
@@ -4,11 +4,11 @@
       <v-col>
         <v-row id="hero">
           <v-col :md="7" :sm="8" cols="12">
-            <h1>Together, we have the power to stop COVID-19.</h1>
+            <h1>The power to stop COVID-19 in the palm of your hand.</h1>
```

Comment:
This is nitpicky, but can we adjust the margin and/or h1 size so that at full-size we don't break on the hyphen in COVID-19?

The power to stop COVID-
19 in the palm of your hand.

vs.

The power to stop COVID-19
in the palm of your hand.

Reply:
Yup, totally fair.  I changed the <h1> `font-size` to 46px from 50px and that resolves it to be the second option you listed (at least on the screen size on my monitor).